test/unit/hpa_vectorized_madvise_large_batch.c

```diff
@@ -195,7 +196,75 @@ TEST_BEGIN(test_vectorized_purge) {
 }
 TEST_END
 
+TEST_BEGIN(test_more_pages_than_batch_page_size) {
```

Comment:
Maybe change the name here as well? While the test moves away from a file where `madvise` was specified, this filename doesn't have such implications so the test name should be more explicit? What about `test_purge_more_than_one_batch_pages`?